src/lib/storage/supabase-storage.ts

```
@@ -0,0 +1,259 @@
+  bucket: string;
+  path: string;
+  metadata?: Record<string, any>;
+}
+
+export class SupabaseStorageService {
+  private static instance: SupabaseStorageService;
+
+  private constructor() {}
+
+  static getInstance(): SupabaseStorageService {
+    if (!this.instance) {
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)